include/dd/Package.hpp

```diff
@@ -2473,22 +2507,22 @@ namespace dd {
             Edge              root{};
             std::stack<Edge*> stack;
 
-            std::unordered_map<decltype(original.p), decltype(original.p)> mapped_node{};
+            std::unordered_map<decltype(original.p), decltype(original.p)> mappedNode{};
 
             Edge* currentEdge = &original;
             if (!currentEdge->isTerminal()) {
-                constexpr std::size_t N = std::tuple_size_v<decltype(original.p->e)>;
+                constexpr std::size_t n = std::tuple_size_v<decltype(original.p->e)>;
                 do {
                     while (currentEdge != nullptr && !currentEdge->isTerminal()) {
-                        for (short i = N - 1; i > 0; --i) {
+                        for (short i = n - 1; i > 0; --i) { // NOLINT(google-runtime-int)
```

Comment:
Why not just use `std::uint16_t` instead of ignoring the Linter warning? Or `std::uint_fast16_t` if you are concerned about the former not being standard mandated (although we are using the fixed width types elsewhere in our projects and, hence, this wouldn't be the first place they are introduced).

Reply:
Good catch. For consistency with the types I'll change `short` to `std::size_t`.